pkg/descheduler/strategies/nodeutilization/highnodeutilization.go

```diff
@@ -29,8 +29,48 @@ import (
 	"sigs.k8s.io/descheduler/pkg/descheduler/evictions"
 	nodeutil "sigs.k8s.io/descheduler/pkg/descheduler/node"
 	podutil "sigs.k8s.io/descheduler/pkg/descheduler/pod"
+	"sigs.k8s.io/descheduler/pkg/utils"
 )
 
+// balanceNodes balance nodes from low utilized nodes to high utilized nodes. A corner case: All nodes are underutilized
+func balanceNodes(sourceNodes, destinationNodes []NodeInfo) ([]NodeInfo, []NodeInfo) {
```

Comment:
Based on the
```
if len(highNodes) == 0 {
	sourceNodes, highNodes = balanceNodes(sourceNodes, highNodes)
}
```
condition len(destinationNodes) will always be 0. Is `balanceNodes` function expected to be used on other places in the future?

Reply:
No, it will only be used in corner case of HighNodeUtilization.